config/containers/logging/local.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Describes how to use the local logging driver.
+keywords: local, docker, logging, driver
+redirect_from:
+- /engine/reference/logging/local/
+- /engine/admin/logging/local/
+title: Local File logging driver
+---
+
+
+The `local` logging driver captures output from container's stdout/stderr and
+writes them to a local file on disk. The format of these files is unspecified.
+By default the `local` driver rotates files once they reach 20MB in size. Each
+rotated file is automatically compressed. The default maximum number of files is
+5, for a total of 100MB (when decompressed) of log data.
+
+The goal of this driver is to be as efficient as possible while still
+remaining consistent. As such log files may not be easily parseable with
```

Comment:
"not easily parseable with standard CLI tools". This needs more explanation as to why.

Reply:
Since this is purely to serve as a warning to users, in which case we should add it as a `Note/Warning`. I prefer `Note`, since it is less alarming.